tests/baselines/reference/api/typescript.d.ts

```diff
@@ -1,21 +1,29 @@
+/*! *****************************************************************************
```

Comment:
The comments in this file are now gone... except for the copyright header, which is now here. Seems wrong?